### DISCUSSION TITLE
What if i use my config prefix to same node/node_modules/npm folder?

### DISCUSSION THREAD

**Discussion by renato-ll** [npm/feedback#516] [Documentation] [answered]
Im on windows...

Ill change my prefix from %AppData%\Roaming\npm to the same folder where i installed nodejs, ie:
to E:\nodejs\roaming-npm\
OK.. way better for me...

BUT what if i take another step:
And put global prefix to the same nodejs/node_modules/npm folder,
I already tested, and it worked fine, 
Is it bad or its ok?

**Comment by ljharb** [comment]
I'm not sure what you're asking. The prefix determines what `npm root -g` will be - ie, where global modules are installed.

imo it's generally better not to change the prefix from the default.

**Reply by renato-ll** [reply]
yes, global in windows is at

%AppData%\Roaming\npm

i moved to

E:\nodejs\roaming-npm
it is common

and i want to move to same npm of nodejs
which is located at:
E:\nodejs\node_modules\npm\

**Comment by MylesBorins** [comment, marked as answer]
Hey @xinxilas . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
